test/integration-test-main.sh

```diff
@@ -397,6 +397,15 @@ function test_external_modification {
 function test_external_creation {
     describe "Test external creation of an object ..."
     local OBJECT_NAME; OBJECT_NAME=$(basename "${PWD}")/"${TEST_TEXT_FILE}"
+
+    # [NOTE]
+    # This test requires that "TEST_TEXT_FILE" does not exist.
+    # (If it exists, it is forgetting to remove it somewhere.)
+    # Also, if noobj_cache is enabled, do the following test first to
+    # ensure that noobj_cache caches it.
```

Comment:
`run_suite` sets a new name for each test so this should always not exist:

```
TEST_TEXT_FILE="test-s3fs-${RANDOM}.txt"
```

Reply:
Some of this comment was unnecessary. It has been deleted.